pebblo_safeloader/langchain/inmemoryloader_postgress/pebblo_safeload.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import os
+
+from dotenv import load_dotenv
+from langchain_community.document_loaders.pebblo import (
+    PebbloSafeLoader,
+    PebbloTextLoader,
+)
+from langchain_openai.embeddings import OpenAIEmbeddings
+from langchain_postgres import PGVector
+from util import get_data
+
+load_dotenv()
+
+PEBBLO_API_KEY = os.getenv("PEBBLO_API_KEY")
+PEBBLO_CLOUD_URL = os.getenv("PEBBLO_CLOUD_URL")
+PG_CONNECTION_STRING = os.getenv("PG_CONNECTION_STRING")
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class PebbloSafeRAG:
+    """
+    Sample app to demonstrate the usage of PebbloSafeLoader
+    using InMemoryLoader and PostgreSQL VectorDB
+    """
+
+    def __init__(self, collection_name: str):
+        self.loader_app_name = "pebblo-safe-loader-in-memory"
```

Comment:
Can you change all references of "in-memory loader" to "text loader"? Including in comments, the dir name, etc.